src/matio.h

```diff
@@ -327,7 +333,9 @@ EXTERN int Mat_VarReadDataLinear(mat_t *mat, matvar_t *matvar, void *data, int s
                                  int edge);
 EXTERN matvar_t *Mat_VarReadInfo(mat_t *mat, const char *name);
 EXTERN matvar_t *Mat_VarReadNext(mat_t *mat);
+EXTERN matvar_t *Mat_VarReadNextPredicate(mat_t *mat, mat_iter_pred_t pred, const void *user_data);
 EXTERN matvar_t *Mat_VarReadNextInfo(mat_t *mat);
+EXTERN matvar_t *Mat_VarReadNextInfoPredicate(mat_t *mat, mat_iter_pred_t pred, const void *user_data);
```

Comment:
Is it really necessary to change the public API of matio if only relevent for MAT 7.3?

Reply:
`Mat_VarReadNextPredicate` was required in a project where we have matio integrated.
Not the case for `Mat_VarReadNextInfoPredicate` though. I can hide it.

Reply:
Should be symmetric. So I am OK to keep as is.